### DISCUSSION TITLE
Tray icon ordering?

### DISCUSSION THREAD

**Discussion by RChesterton** [Rem0o/FanControl.Releases#2037] [Q&A]
I have set up 2 tray icons - one for CPU, one for GPU.

No matter what I do, I cannot get them to remain in the correct order (CPU first, GPU second).  They reset after each reboot.

I have tried adding them into the settings in different orders, manually moving them in the tray.

Any ideas?

**Comment by k9education** [comment]
I don't think it's possible to lock them in place.  I have the same issue regardless of which monitoring software I use.  It's irritating AF, but seems to be the fault of Windows.

**Reply by RChesterton** [reply]
I would think it depends on the order the app loads them in. 

It seems consistent. Unfortunately, it’s consistently wrong 😂

**Reply by k9education** [reply]
Mine move periodically without even rebooting the system or restarting the app.  Same deal with CoreTemp and HWiNFO.  I'm a little OCD about these things so it bugs me. :p

**Reply by RChesterton** [reply]
That's annoying lol

Mine is just consistently the wrong way around.  I want CPU first, GPU second.  Every time it starts up, it is the other way around.  I move them and they stay until I reboot/logout and in again.. and then they're back to the other way.

**Reply by wayneguk** [reply]
I put GPU on top and then CPU. In tray they order CPU first then GPU. On restart they move opposite. Rinse and repeat.

**Reply by Rem0o** [reply]
@wayneddgu  you can't rely on the reordering you do in windows, it won't hold.  If you want CPU first, put CPU first in FanControl.

**Reply by wayneguk** [reply]
Hi @Rem0o  I'm not doing it in windows I'm doing it in FanControl but it always swaps them on restart. I've gone back to v190

**Reply by Rem0o** [reply]
@wayneddgu  Windows 10 or 11 ?  In 11 I tested on multiple computers, all work as expected:
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/4887d1c8-4ff2-4360-805b-08324c20c16e)

**Comment by DexyDexy** [comment]
I have found a way to make all FanControl icons fully independent and movable in tray.

1. Open regedit and go to Computer\HKEY_CURRENT_USER\Control Panel\NotifyIconSettings you will notice many subkeys with numeric names. Check every one and find one which points to FanControl without any name of sensor.
2. Delete this numeric named key (looks like folder in regedit).
3. Sign out - sign in
4. Repeat untill all FanControl Icons are moveble and dont "stick" to each other anymore
5. enjoy

**Reply by godzinilla** [reply]
> I have found a way to make all FanControl icons fully independent and movable in tray.
> 
> 1. Open regedit and go to Computer\HKEY_CURRENT_USER\Control Panel\NotifyIconSettings you will notice many subkeys with numeric names. Check every one and find one which points to FanControl without any name of sensor.
> 2. Delete this numeric named key (looks like folder in regedit).
> 3. Sign out - sign in
> 4. Repeat untill all FanControl Icons are moveble and dont "stick" to each other anymore
> 5. enjoy

It is not working for me but I have removed all the entries at once, then signed out/on.
